cmd/conf/chain.go

```diff
@@ -4,28 +4,35 @@
 package conf
 
 import (
+	"time"
+
 	"github.com/offchainlabs/nitro/cmd/genericconf"
+	"github.com/offchainlabs/nitro/util/rpcclient"
 	flag "github.com/spf13/pflag"
 )
 
 type L1Config struct {
-	ChainID            uint64                   `koanf:"chain-id"`
-	URL                string                   `koanf:"url"`
-	ConnectionAttempts int                      `koanf:"connection-attempts"`
-	Wallet             genericconf.WalletConfig `koanf:"wallet"`
+	ChainID    uint64                   `koanf:"chain-id"`
+	Connection rpcclient.ClientConfig   `koanf:"connection" reload:"hot"`
+	Wallet     genericconf.WalletConfig `koanf:"wallet"`
+}
+
+var L1ConnectionConfigDefault = rpcclient.ClientConfig{
+	URL:            "",
+	Retries:        2,
+	Timeout:        time.Minute * 5,
```

Comment:
We don't do any unbounded log queries, so I think it's safe to shorten this quite a lot. I think 1 minute would be a reasonable default.

Reply:
One thing to note is that DAS queries have its own timeout which should be larger so that this timeout hits and retries first.